.editorconfig

```
@@ -0,0 +1,447 @@
+dotnet_naming_rule.local_var_and_method_args.symbols = local_var_and_method_args
+dotnet_naming_rule.local_var_and_method_args.style = camel_case
+
+dotnet_naming_rule.private_member_variables.severity = error
+dotnet_naming_rule.private_member_variables.symbols = private_member_variables
+dotnet_naming_rule.private_member_variables.style = pascal_case_with_underscore_prefix
+
+# Symbol specifications
+
+dotnet_naming_symbols.interface.applicable_kinds = interface
+dotnet_naming_symbols.interface.applicable_accessibilities = public, internal, private, protected, protected_internal, private_protected
+dotnet_naming_symbols.interface.required_modifiers =
+
+dotnet_naming_symbols.const_field_members.applicable_kinds = field
+dotnet_naming_symbols.const_field_members.applicable_accessibilities = public, internal, private, protected, protected_internal, private_protected
+dotnet_naming_symbols.const_field_members.required_modifiers = const
+
+dotnet_naming_symbols.types.applicable_kinds = class, struct, interface, enum
+dotnet_naming_symbols.types.applicable_accessibilities = public, internal, private, protected, protected_internal, private_protected
+dotnet_naming_symbols.types.required_modifiers =
+
+dotnet_naming_symbols.non_field_members.applicable_kinds = property, event, method
+dotnet_naming_symbols.non_field_members.applicable_accessibilities = public, internal, private, protected, protected_internal, private_protected
+dotnet_naming_symbols.non_field_members.required_modifiers =
+
+dotnet_naming_symbols.private_member_variables.applicable_kinds = field
+dotnet_naming_symbols.private_member_variables.applicable_accessibilities = private, private_protected
+dotnet_naming_symbols.private_member_variables.required_modifiers =
+
+dotnet_naming_symbols.local_var_and_method_args.applicable_kinds = parameter, local
+
+# Naming styles
+
+dotnet_naming_style.begins_with_i.required_prefix = I
+dotnet_naming_style.begins_with_i.required_suffix =
+dotnet_naming_style.begins_with_i.word_separator =
+dotnet_naming_style.begins_with_i.capitalization = pascal_case
+
+dotnet_naming_style.pascal_case.required_prefix =
+dotnet_naming_style.pascal_case.required_suffix =
+dotnet_naming_style.pascal_case.word_separator =
+dotnet_naming_style.pascal_case.capitalization = pascal_case
+
+dotnet_naming_style.pascal_case_with_underscore_prefix.required_prefix = _
+dotnet_naming_style.pascal_case_with_underscore_prefix.required_suffix =
+dotnet_naming_style.pascal_case_with_underscore_prefix.word_separator =
+dotnet_naming_style.pascal_case_with_underscore_prefix.capitalization = pascal_case
+
+dotnet_naming_style.camel_case.required_prefix =
+dotnet_naming_style.camel_case.required_suffix =
+dotnet_naming_style.camel_case.word_separator =
+dotnet_naming_style.camel_case.capitalization = camel_case
+
+dotnet_style_operator_placement_when_wrapping = beginning_of_line
+tab_width = 4
+indent_size = 4
+end_of_line = crlf
+indent_style = space
+max_line_length = 150

```

Can you reduce the maximum line length? Let's say 100, which is much easier for side-by-side code reviews.

```suggestion
max_line_length = 100
```